data/json/items/gun/223.json

```
@@ -292,13 +196,89 @@
       }
     ]
   },
+  {
+    "//": "First group change from 'NATO Assault Rifle'. Groups all 'H&K416' related rifles together, alongside the SCAR-L due to its similar properties. Used M27 due to it being the most popular of the four in the US.",
+    "id": "m27_assault_rifle",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "M27 Assault Rifle" },
```

Item names are in lower case.
```suggestion
    "name": { "str": "M27 assault rifle" },
```